code/modules/mob/living/login.dm

```diff
@@ -29,4 +29,6 @@
 	//Should update regardless of if we can ventcrawl, since we can end up in pipes in other ways.
 	update_pipe_vision()
 
+	walk(src, 0) // if mob is moving under ai control, then stop AI movement
```

Comment:
Are xenos trying to run away? Are brains trying to run away?  Are human mobs running around like chickens with their heads cut off? What is OOP?

This has no reason to be in the login for all `/living` when the problem is very clearly only associated with simple mobs.

Reply:
Moved to simple_animal/Login()